### ISSUE TITLE
Example set up on each js framework? 

### ISSUE THREAD

**Issue by goldcoders** [swc-project/swc#1090]
I find the documentation quite short...
Mainly in implementing on different kind of 
Frameworks such as react, Vue, svelte, angular. 

I cant find any also in the Web any examples. 

So Ive posted here if you can give each an example?

I really would love to test how it performs on svelte with typescript...

Thanks

**Comment by Phaqui** [comment]
Agreed. I am in the same situation of wanting to speed up Svelte+TypeScript, and using Rollup as my bundler (which is the default for Svelte). Waiting 5-8 seconds for each round of every tiny change to be visible gets really tiresome after a while...

I did get the <script lang="ts"> blocks to compile using swc instead of tsc with a change to the rollup config, but replacing compilation of .ts files themselves was not as straight forward.

I have not tried to use the Spack bundler to bundle svelte+typescript, but I really don't know if I want to even try, at this point.

**Comment by jpike88** [comment]
@kdy1 this is also a blocker to adoption, lack of examples. I also don't know if it's even feasible to build an angular app with this.

Is this really only for nodejs projects?

**Comment by kdy1** [comment]
@jpike88 It's a library both for web apps and node js apps. It's close to a compiler for web, because node js implements lots of recent ecmascript spec.

**Comment by jpike88** [comment]
Alright, my next questions would be:

- does a minimal repo/example exist for this? I'll try to stab in dark later today but that's always good to have.

- what about the bundling or assets (css, images etc)?

**Comment by kdy1** [comment]
> does a minimal repo/example exist for this? I'll try to stab in dark later today but that's always good to have.

Not at the moment.

> what about the bundling of assets (css, images etc)?

This is in my todo list, but it's not supported yet.

> does this handle hot reloading (even just an auto page refresh)? Is that on the roadmap?

Not supported yet but definitely in roadmap for the bundler.

**Comment by jpike88** [comment]
Cool, thanks for your prompt response.

I’ll do my best to fire it up against my angular project this weekend and report issues as I see them.

**Comment by wuyudi** [comment]
Here is a demo svelte project with swc, swc is blazing fast :+1:
https://github.com/wuyudi/svelte-swc

**Comment by kwonoj** [comment]
Going to close this one, while it'd be good to have some doc enhancement this is very wide topic not clearly actionable. swc's doc provides the usage for the swc/core itself, and for the surrdoing ecosystem tools each may have corresponding config / docs for their own. We'll accept suggestions for the doc if swc/core related docs need some more touch.
